modules/javafx.graphics/src/main/java/com/sun/javafx/tk/Toolkit.java

```diff
@@ -414,11 +416,15 @@ public void firePulse() {
         @SuppressWarnings("removal")
         final Map<TKPulseListener,AccessControlContext> postScenePulseList =
                 new WeakHashMap<>();
+        final Map<TKPulseListener,AccessControlContext> cleanupList =
+                new WeakHashMap<>();
```

Comment:
I wonder why we need `WeakHashMap` here. These maps are short-lived anyway, since they're local variables.

Reply:
I've changed it now to a normal HashMap.